crates/proto/src/rr/record_type.rs

```diff
@@ -149,7 +149,7 @@ impl FromStr for RecordType {
     /// ```
     fn from_str(str: &str) -> ProtoResult<Self> {
         // TODO missing stuff?
-        match str {
+        match str.to_uppercase().as_str() {
```

Comment:
Similar comment to above, but instead here: https://github.com/bluejekyll/trust-dns/blob/main/crates/client/src/serialize/txt/zone.rs#L224